docs/docs/diff.md

```diff
@@ -0,0 +1,65 @@
+# Differences with bellard/quickjs
+
+This project aims to be a drop-in replacement for those already using QuickJS.
+Minimal API changes might be necessary.
```

Comment:
We should probably flesh those out. The only one that comes to mind is class ids may need tweaking.

Reply:
- The malloc function changes
- SetOpaque can fail

Though those 2 are rare to see and maybe belong in advanced use cases...

Reply:
On my next pass I'll try and add some "migration" section.